src/components/Page.jsx

```diff
@@ -1,21 +1,27 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
+import { withRouter } from 'react-router-dom';
 import Subnav from './Subnav.jsx';
 import styles from '../app.scss';
 import actions from '../redux/app/actions';
 import withLoader from '../hoc/withLoader';
+import biblioLinkChecker from '../lib/pages/biblioLinkChecker';
 
-const Page = ({ path, subnavPath, content, menu, useHash }) => (
+const Page = ({ path, subnavPath, content, menu, useHash, history }) => (
   <div className={styles.about}>
     <div className={styles.container}>
       <div className={styles.head}>
         <Subnav menu={menu} basePath={subnavPath} useHash={useHash} />
       </div>
       <header>{content.title}</header>
-      <div className={styles.body} id="about">
+      <div className={styles.body} id='about'>
         {content.body.map((para, idx) => (
-          <p key={idx} dangerouslySetInnerHTML={{ __html: para }} />
+          <p
+            key={idx}
+            dangerouslySetInnerHTML={{ __html: para }}
+            onClick={e => biblioLinkChecker({ e, history })}
```

Comment:
what is the purpose of this? in general, only clickable elements in the HTML sense (buttons and links) should have an onClick event